firefox-ios/Client/Ecosia/UI/NTP/MultiPurposeHeader/NTPMultiPurposeEcosiaHeader.swift

```diff
@@ -78,23 +74,13 @@ struct NTPMultiPurposeEcosiaHeaderView: View {
     var body: some View {
```

Comment:
Shouldn't we add `ecosiaThemed(windowUUID, $theme)`, create a theme and use it on the button? Saw the old way was removed but not sure how it is set now here 🤔

Reply:
By default, SwiftUI doesn't render any background or colors applied to `View`s (unless Text or some other specific cases).
That is super convenient for us, as we let the top-most view behave with its "transparent/clear" default behaviour, and we only let the specific view (`EcosiaAISearchButton`) react to theme changes.